src/api/review.js

```diff
@@ -0,0 +1,21 @@
+const API_BASE_URL = import.meta.env.VITE_API_BASE_URL;
+
+export const getReview = async () => {
+  try {
+    const response = await fetch(`${API_BASE_URL}/review`);
+    const data = await response.json();
+    const reviewData = data.data;
+    const newData = reviewData.map((review) => {
+      return {
+        title: review.title + " (new)",
```

Comment:
can we please remove this + new and map?

I did that because I wanted to add more data in our fetched array to play around with the slider

you should `return reviewData` right away

Reply:
![udah mbak](https://github.com/project-zusammen/hidden-gem-frontend/assets/121326593/26886164-4b37-4da3-bcee-5c499cc369d4)

udah mbak

Reply:
I don't see your updated changes, can you please push your latest commit?